### PR TITLE
Add presubmit jobs for knmstate release-0.52

### PR DESCRIPTION
**Pull request by rhrazdil**
Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits-0.52.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "NMSTATE_PIN=future automation/check-patch.e2e-k8s.sh"
```
Comment (qinqon) [inline github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits-0.52.yaml:90]: We don't need the fufure lane on release branches

**Review by qinqon — CHANGES_REQUESTED** [review]
Let's remove future job

File: github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits-0.52.yaml (hunk 1 above)
Comment (rhrazdil) [reply github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits-0.52.yaml]: Removed

**Comment by qinqon** [comment]
/lgtm
/approve
/hold
Let's wait for rehearse result

**Comment by fgimenez** [comment]
/override rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s-0.52

**Pull request merged**